### PR TITLE
Updated husky to fix issue with ckeditor plugins

### LINKED ISSUE #3313: Trying to display buttons for text and background color in Sulu ckeditor
Greetings everyone,

| Q | A
| --- | ---
| Bug? | no
| New Feature? | no
| Sulu Version | 1.5
| Browser Version | Chrome

#### Actual Behavior

I'm trying to display ckeditor buttons to enable text color and background color customizations.
When setting the config, the buttons for text color and background color don't appear.

#### Expected Behavior

They should appear (see https://github.com/sulu/sulu/issues/864#issuecomment-165091811).

#### Steps to Reproduce
It looks like you have files for this plugin in `vendor/sulu/sulu/src/Sulu/Bundle/AdminBundle/Resources/public/js/vendor/husky/vendor/ckeditor/plugins/colorbutton`.

`app/Resources/views/templates/homepage.html.twig`
```
<?xml version="1.0" ?>
<template xmlns="http://schemas.sulu.io/template/template"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd">

    <key>homepage</key>

    <view>templates/homepage</view>
    <controller>SuluWebsiteBundle:Default:index</controller>
    <cacheLifetime>2400</cacheLifetime>

    <meta>
        <title lang="en">Homepage</title>
        <title lang="de">Startseite</title>
    </meta>

    <properties>
        <section name="highlight">
            <properties>
                <property name="title" type="text_line" mandatory="true">
                    <meta>
                        <title lang="en">Title</title>
                        <title lang="de">Titel</title>
                    </meta>
                    <params>
                        <param name="headline" value="true"/>
                    </params>

                    <tag name="sulu.rlp.part"/>
                </property>

                <property name="url" type="resource_locator" mandatory="true">
                    <meta>
                        <title lang="en">Resourcelocator</title>
                        <title lang="de">Adresse</title>
                    </meta>

                    <tag name="sulu.rlp"/>
                </property>
            </properties>
        </section>

        <property name="article" type="text_editor">
            <meta>
                <title lang="en">Article</title>
                <title lang="de">Artikel</title>
            </meta>

            <params>
                <param name="table" value="true"/>
                <param name="link" value="true"/>
                <param name="paste_from_word" value="true"/>
                <param name="height" value="100"/>
                <param name="max_height" value="200"/>
                <!-- CKEditor Parameters examples: -->
                **<param name="custom_config" value="plugins/colorbutton/plugin.js" />
                <param name="toolbar" value='[{"name":"colors","items":["TextColor","BGColor"]}]' />
                <param name="extra_plugins" value="justify,format,sourcearea,link,table,pastefromword,autogrow,colorbutton" />**
            </params>
        </property>
    </properties>
</template>
```

Thanks for the help :)

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #3313 
| Related issues/PRs | https://github.com/massiveart/husky/pull/770
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR updates husky library.

### CODE REVIEW

**Comment by danrot** [comment]
@wachterjohannes Seems like there is a [unstable test](https://travis-ci.org/sulu/sulu/jobs/232791963#L764). I know it is not related to this PR, but since it came up here, it would be nice if you could fix it as well.

**Comment by wachterjohannes** [comment]
@danrot done

File: src/Sulu/Bundle/CustomUrlBundle/Tests/Functional/Controller/CustomUrlControllerTest.php [hunk 1]
```diff
@@ -711,8 +711,8 @@ public function testCGet($items)
             $this->assertArrayHasKey('creator', $responseData);
             $this->assertArrayHasKey('changer', $responseData);
 
-            $this->assertEquals($requestTime, new \DateTime($responseData['created']), '', 2);
-            $this->assertEquals($requestTime, new \DateTime($responseData['changed']), '', 2);
+            $this->assertEquals($requestTime, new \DateTime($responseData['created']), '', 5);
+            $this->assertEquals($requestTime, new \DateTime($responseData['changed']), '', 5);
```
Comment (danrot) [inline src/Sulu/Bundle/CustomUrlBundle/Tests/Functional/Controller/CustomUrlControllerTest.php]: Is it really fixed that way? Haven't we already used `assertGreaterThanOrEqual` in other places, to make sure the test will always succeed, no matter how long the process might hang before?

**Comment by wachterjohannes** [comment]
@danrot i have changed that to `assertLessThanOrEqual`.

**Pull request merged**